bleach/__init__.py

```diff
@@ -347,6 +347,13 @@ def link_repl(match):
         if url.startswith('('):
             _wrapping = strip_wrapping_parentheses(url)
             url, open_brackets, close_brackets = _wrapping
+        elif url.endswith('/)'):
+            # This is a very common corner case where the URL is listed
+            # last within a longer text within parentheses.
+            # Example: (check my site at http://foo.bar/)
+            # Do not consider the closing ) as part of the URL.
+            url, open_brackets, close_brackets = url[:-1], 0, 1
```

Comment:
Sorry about that, PyCharm did that automatically and I did not realize that it reformatted lines. 

We had this discussion before and it is just another example how using flake8 and its ilk to enforce the code "quality" makes life more miserable for everyone. Basically it is counterproductive.

Tests kept failing because the buildbot would enforce rules for code not being formatted right, but then when one uses a formatter that is supposedly pep8 aware and makes these errors go away it ends up reformatting the code with rules that flake8 did not enforce. 

By the time of the third attempt to fix failing test that had nothing to do with code quality I lost the focus of ensuring that the changes are minimal and just wanted to shut it up already.